pkgs/development/compilers/orc/default.nix

```diff
@@ -1,23 +1,31 @@
-{ stdenv, fetchurl }:
-
-stdenv.mkDerivation rec {
-  name = "orc-0.4.29";
+{ stdenv, fetchurl, meson, ninja
+, gtk-doc ? null, file, docbook_xsl
+, buildDevDoc ? gtk-doc != null
+}: let
+  inherit (stdenv.lib) optional optionals;
+in stdenv.mkDerivation rec {
+  pname = "orc";
+  version = "0.4.32";
 
   src = fetchurl {
-    url = "https://gstreamer.freedesktop.org/src/orc/${name}.tar.xz";
-    sha256 = "1cisbbn69p9c8vikn0nin14q0zscby5m8cyvzxyw2pjb2kwh32ag";
+    url = "https://gstreamer.freedesktop.org/src/orc/${pname}-${version}.tar.xz";
+    sha256 = "1w0qmyj3v9sb2g7ff39pp38b9850y9hyy0bag26ifrby5f7ksvm6";
   };
 
-  outputs = [ "out" "dev" ];
+  outputs = [ "out" "dev" ]
+     ++ optional buildDevDoc "devdoc"
```

Comment:
Isn't this normally named doc?

Reply:
`doc` also exists (and is more common), but `devdoc` is one of the other "magic" output names in nixpkgs: https://nixos.org/manual/nixpkgs/stable/#multiple-output-file-type-groups , namely gtk-doc puts its stuff there if it exists, otherwise it's completely removed later in the build.